packages/techdocs-common/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@backstage/techdocs-common",
+  "description": "Common functionalities for TechDocs, to be shared between techdocs plugins and techdocs-cli",
+  "version": "0.1.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "private": false,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "packages/techdocs-common"
+  },
+  "keywords": [
+    "techdocs",
+    "backstage"
+  ],
+  "license": "Apache-2.0",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "backstage-cli build --outputs cjs,types",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "bugs": {
+    "url": "https://github.com/backstage/backstage/issues"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.3.3",
+    "@backstage/catalog-model": "^0.4.0",
+    "@backstage/config": "^0.1.1",
+    "@google-cloud/storage": "^5.6.0",
+    "@types/dockerode": "^3.2.1",
+    "@types/express": "^4.17.6",
+    "cross-fetch": "^3.0.6",
+    "dockerode": "^3.2.1",
+    "express": "^4.17.1",
+    "fs-extra": "^9.0.1",
+    "git-url-parse": "^11.4.0",
+    "js-yaml": "^3.14.0",
+    "klaw": "^3.0.0",
```

Comment:
You could use recursive-readdir here instead. It's being used in a few other places in backstage.

Reply:
@hooloovooo Yes! `klaw` was not working with `mock-fs`. Switched to `recursive-readdir` and it's awesome! Smaller code plus supports `mock-fs` out-of-the-box.